### PR TITLE
Float double formatting upd

### PR DESCRIPTION
**Pull request by D1-3105**
Check exec.c & exec.h

### CODE REVIEW

File: ZRC_evm/exec.h [hunk 1]
```diff
@@ -10,12 +10,16 @@
...
-	#define UnarOperationString "\nUnary operation:\n\n"
+	#define UnarOperationString "\nUnary operations:\n\n"
 #endif
 
 #ifndef BinaryOperationString
-	#define BinaryOperationString "\nBinary operation:\n\n"
+	#define BinaryOperationString "\nBinary operations:\n\n"
 #endif
+#ifndef TriganometricOperationString
+	#define TriganometricOperationString "\nTriganometric operations:\n\n"
```
Comment (nikoloskorzun) [inline ZRC_evm/exec.h:20]: Why?

File: ZRC_evm/exec.c [hunk 2]
```diff
@@ -1205,50 +1211,64 @@ void perform_unsigned_integers()
...
 
+    // unar
+    printf(UnarOperationString);
+    timeit(sqrt_floating, "FSQRT REG", 2, NumberOfElementsArray, x_f);
+    timeit(l2_floating, "FYL2X REG", 2, NumberOfElementsArray, x_f);
+    timeit(ln_floating, "LN REG", 2, NumberOfElementsArray, x_f);
 
-    // erase Float
+    // trigan
+    printf(TriganometricOperationString);
```
Comment (nikoloskorzun) [inline ZRC_evm/exec.c:1241]: Triganometric not unary? NO.

File: ZRC_evm/exec.c [hunk 3]
```diff
@@ -1259,30 +1279,63 @@ void perform_floats() {
...
-    timeit(tan_double, "TAN OF DOUBLE", 2, NumberOfElementsArray, x_d);
-    timeit(atan_double, "ATAN OF DOUBLE", 2, NumberOfElementsArray, x_d);
-    //mem
-    printf("double MEMORY [Double 64 bit]\n");
-    timeit(add_mem_double, "ADD OF DOUBLE", 3, NumberOfElementsArray, x_d, y_d);
-    timeit(sub_mem_double, "SUB OF DOUBLE", 3, NumberOfElementsArray, x_d, y_d);
-    timeit(mul_mem_double, "MUL OF DOUBLE", 3, NumberOfElementsArray, x_d, y_d);
-    timeit(div_mem_double, "DIV OF DOUBLE", 3, NumberOfElementsArray, x_d, y_d);
+
+    printf(TriganometricOperationString);
```
Comment (nikoloskorzun) [inline ZRC_evm/exec.c:1297]: ...

File: ZRC_evm/exec.c [hunk 4]
```diff
@@ -1259,30 +1279,63 @@ void perform_floats() {
...
+    timeit(mul_mem_double, "MUL REG, MEM", 3, NumberOfElementsArray, x_d, y_d);
+    timeit(div_mem_double, "DIV REG, MEM", 3, NumberOfElementsArray, x_d, y_d);
 
     free(x_d);
     free(y_d);
 }
 
 
+void perform_floats() {
+    printf(DelimString);
```
Comment (nikoloskorzun) [inline ZRC_evm/exec.c:1328]: printf's move to *_reg() and *_mem() functions

File: ZRC_evm/exec.h (hunk 1 above)
Comment (D1-3105) [reply ZRC_evm/exec.h:20]: Trigan operations are separated part of math :O

**Review by nikoloskorzun — APPROVED** [review]

**Pull request merged**